diff --git a/veridian-slang/build.rs b/veridian-slang/build.rs
index fbc4c27..2e03188 100644
--- a/veridian-slang/build.rs
+++ b/veridian-slang/build.rs
@@ -4,6 +4,7 @@ use std::fs::File;
 use std::path::PathBuf;
 use tar::Archive;
 
+#[allow(dead_code)]
 fn download_slang() -> Result<(), Box<dyn std::error::Error>> {
     let target = "https://github.com/MikePopoloski/slang/releases/download/v0.7/slang-linux.tar.gz";
     let fname = "slang-linux.tar.gz";
@@ -34,8 +35,6 @@ fn main() {
     println!("cargo:rerun-if-changed=slang_wrapper/src/slang_wrapper.h");
     println!("cargo:rerun-if-changed=slang_wrapper/src/slang_lib.cpp");
 
-    download_slang().unwrap();
-
     build_slang_wrapper();
 
     let bindings = bindgen::Builder::default()
